### DISCUSSION TITLE
I saw that llama.cpp has an OpenAI compatible server now

### DISCUSSION THREAD

**Discussion by Madd0g** [withcatai/node-llama-cpp#118] [Q&A] [answered]
Can we pass a flag to start it?

**Comment by giladgd** [comment, marked as answer]
The server example in `llama.cpp` is only an example and is not meant for production use, nor is it exposed as part of the `llama.cpp` stable interface.
It also doesn't implement significant parts of OpenAI API.

An inference server is planned as part of the [roadmap](https://github.com/orgs/withcatai/projects/1), but it'll take some time.
The next version (#105) will include many improvements required to implement a sufficient API for most use cases people are using OpenAI API for.
